Cython/Includes/cpython/mem.pxd

```diff
@@ -1,3 +1,10 @@
+from cpython.version cimport PY_VERSION_HEX
+
+cdef extern from *:
```

Comment:
Just keep these together with the others below. They are all defined in `Python.h`.